src/components/VIconButton/VIconButton.vue

```diff
@@ -26,23 +26,11 @@ import VIcon, { type IconProps } from '~/components/VIcon/VIcon.vue'
 import VButton, { type ButtonProps } from '~/components/VButton.vue'
 
 const SIZE_MAP = Object.freeze({
-  tiny: { icon: 'w-6 h-6', button: 'w-6 h-6' },
-  small: { icon: 'w-6 h-6', button: 'w-10 h-10' },
-  smaller: { icon: 'w-6 h-6', button: 'w-8 h-8' },
-  // TODO: audit the sizes after new header is done.
-  'new-small': { icon: 'w-4 h-4', button: 'w-8 h-8' },
-  'search-small': {
-    icon: 'w-6 h-6',
-    button: 'w-10 md:w-12 h-10 md:h-12',
-  },
-  'search-medium': { icon: 'w-6 h-6', button: 'w-12 h-12' },
-  'search-large': { icon: 'w-6 h-6', button: 'w-14 h-14' },
-  'search-standalone': {
-    icon: 'w-6 h-6',
-    button: 'w-14 md:w-[69px] h-14 md:h-[69px]',
-  },
-  medium: { icon: 'w-8 h-8', button: 'w-14 h-14' },
-  large: { icon: 'w-12 h-12', button: 'w-20 h-20' },
+  tiny: { icon: 6, button: 'w-6 h-6' },
+  small: { icon: 6, button: 'w-10 h-10' },
+  medium: { icon: 6, button: 'w-12 h-12' },
+  larger: { icon: 8, button: 'w-14 h-14' },
+  large: { icon: 12, button: 'w-20 h-20' },
```

Comment:
Just a nit pick on the names here: In my head I expected `larger` to be bigger than `large` :thinking: Maybe `larger -> large` and `large -> extra-large` or something would be clearer in the "order" of how big one or the other is?

Reply:
Thank you for helping with the naming :)